Unable to push report to SonarQube server when integrating with dependencyCheck plugin

## Describe the bug

I'm currently using the Gradle plugin as:

```groovy
import org.owasp.dependencycheck.reporting.ReportGenerator

plugins {
  id "org.owasp.dependencycheck" version "5.3.0"
  id "org.sonarqube" version "2.7"
}

dependencies() {
    compileOnly("org.owasp:dependency-check-gradle:5.3.0")
}

dependencyCheck {
  analyzers {
    assemblyEnabled = false
    nodeEnabled = false
  }
  failOnError = false
  formats = [ReportGenerator.Format.HTML, ReportGenerator.Format.XML]
}

sonarqube {
  properties {
    property "sonar.dependencyCheck.htmlReportPath", "build/reports/dependency-check-report.html"
    property "sonar.dependencyCheck.reportPath", "build/reports/dependency-check-report.xml" // <- Big time with this one
    property "sonar.dependencyCheck.xmlReportPath", "build/reports/dependency-check-report.xml"
    property "sonar.projectDescription", "${project.description}"
    property "sonar.projectKey", "tld.domain:${project.name}"
    property "sonar.projectName", "tld.domain::${project.name}"
  }
}
```

I therefore execute this as: `./gradlew -Dsonar.host.url=https://reports.domain.tld/sonarqube -Dsonar.login=token dependencyCheckAnalyze sonarqube --info`. Everything is fine until the point it's trying to upload the results to the server:

```shell
> Task :sonarqube FAILED
Sensor JavaSquidSensor [java] (done) | time=220668ms
Sensor PmdSensor [pmd]
Sensor PmdSensor [pmd] (done) | time=1ms
Sensor SurefireSensor [java]
parsing [/path/to/project/build/test-results/test]
Sensor SurefireSensor [java] (done) | time=62ms
Sensor Removed properties sensor [java]
Sensor Removed properties sensor [java] (done) | time=1ms
Sensor JavaXmlSensor [java]
Sensor JavaXmlSensor [java] (done) | time=3ms
Sensor HTML [web]
Sensor HTML [web] (done) | time=6ms
Sensor Dependency-Check [dependencycheck]
Process Dependency-Check report
Process Dependency-Check report (done) | time=139ms
:sonarqube (Thread[Daemon worker,5,main]) completed. Took 3 mins 53.184 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sonarqube'.
> java.lang.NullPointerException (no error message)
```

I know for sure that this setting: `property "sonar.dependencyCheck.reportPath", "build/reports/dependency-check-report.xml"` is what's causing that `java.lang.NullPointerException (no error message)`. On the other hand, if I don't use it, the action is successfully executed, but nothing is uploaded to the SonarQube server.

## To Reproduce

Steps to reproduce the behavior:
1. Grab one of the example projects with Gradle
2. Modify the `dependencycheck` version to `5.3.0` and the SonarQube plugin version to `2.7`
3. Execute `dependencyCheckAnalyze` and `sonarqube` tasks with the options to identify the location to a SonarQube server and the token needed to upload the result(s)

## Current behavior

The action(s) fail with a `java.lang.NullPointerException (no error message)`.

## Expected behavior

To be able to complete the report(s) and upload the results successfully.

## Versions (please complete the following information)

-  dependency-check: `5.3.0`
-  sonarqube: `2.7`
-  dependency-check-sonar-plugin: `5.3.0`

--- a/examples/multi-module-maven/pom.xml
+++ b/examples/multi-module-maven/pom.xml
@@ -54,7 +54,7 @@
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>5.2.2</version>
+                    <version>5.3.0</version>
                     <configuration>
                         <format>ALL</format>
                     </configuration>

--- a/examples/multi-project-csproj/Dockerfile
+++ b/examples/multi-project-csproj/Dockerfile
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/dotnet/core/sdk:2.2
 ARG sonarHost
 
 ENV NODEREPO=node_10.x \
-    DEPENDENCY_CHECK=5.2.1 \
+    DEPENDENCY_CHECK=5.3.0 \
     PATH="$PATH:/root/.dotnet/tools"
 
 RUN /usr/bin/apt-get update && \
@@ -12,7 +12,7 @@ RUN /usr/bin/apt-get update && \
     dotnet tool install --global dotnet-sonarscanner && \
     # Install dependency-check
     /usr/bin/wget -q -O /opt/dependency-check-${DEPENDENCY_CHECK}-release.zip https://dl.bintray.com/jeremy-long/owasp/dependency-check-${DEPENDENCY_CHECK}-release.zip && \
-    /usr/bin/wget -q -O /opt/dependency-check-${DEPENDENCY_CHECK}-release.zip.asc https://dl.bintray.com/jeremy-long/owasp/dependency-check-5.2.1-release.zip.asc && \
+    /usr/bin/wget -q -O /opt/dependency-check-${DEPENDENCY_CHECK}-release.zip.asc https://dl.bintray.com/jeremy-long/owasp/dependency-check-${DEPENDENCY_CHECK}-release.zip.asc && \
     /usr/bin/gpg --batch --keyserver keyserver.ubuntu.com --recv-keys F9514E84AE3708288374BBBE097586CFEA37F9A6 && \
     /usr/bin/gpg --batch --verify /opt/dependency-check-${DEPENDENCY_CHECK}-release.zip.asc /opt/dependency-check-${DEPENDENCY_CHECK}-release.zip && \
     /usr/bin/unzip /opt/dependency-check-${DEPENDENCY_CHECK}-release.zip -d /opt && \

--- a/examples/single-module-gradle/build.gradle
+++ b/examples/single-module-gradle/build.gradle
@@ -3,7 +3,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'org.owasp:dependency-check-gradle:5.2.2'
+        classpath 'org.owasp:dependency-check-gradle:5.3.0'
     }
 }
 

--- a/examples/single-module-kotlin-dsl-gradle/build.gradle.kts
+++ b/examples/single-module-kotlin-dsl-gradle/build.gradle.kts
@@ -4,7 +4,7 @@ plugins {
     application
     kotlin("jvm") version "1.3.50"
     id("org.sonarqube") version "2.8"
-    id("org.owasp.dependencycheck") version "5.2.2"
+    id("org.owasp.dependencycheck") version "5.3.0"
 }
 
 repositories {
@@ -13,7 +13,7 @@ repositories {
 
 dependencies {
     compile("com.google.guava:guava:21.0")
-    compile("org.owasp:dependency-check-gradle:5.2.2")
+    compile("org.owasp:dependency-check-gradle:5.3.0")
     compile("org.springframework:spring:2.0")
     testCompile("junit:junit:4.12")
     implementation(kotlin("stdlib-jdk8"))

--- a/examples/single-module-maven/pom.xml
+++ b/examples/single-module-maven/pom.xml
@@ -71,7 +71,7 @@
             <plugin>
                 <groupId>org.owasp</groupId>
                 <artifactId>dependency-check-maven</artifactId>
-                <version>5.2.2</version>
+                <version>5.3.0</version>
                 <configuration>
                     <format>ALL</format>
                 </configuration>

--- a/examples/single-module-nodejs/README.md
+++ b/examples/single-module-nodejs/README.md
@@ -6,7 +6,7 @@ Integrates [Dependency-Check][] analysis and reporting into SonarQube v6.7 or hi
 
 -   [npm][]
 -   [dependency-check-cli][]
--   [sonar-runner][]
+-   [sonar-scanner][]
 
 ## Usage
 
@@ -17,8 +17,8 @@ npm install
 mkdir -p reports/dependency-check
 # Run Dependency-Check
 dependency-check.sh --format ALL -s . --out reports/dependency-check --project "nodejs example"
-# Invoke Sonar-Runner. This reads from sonar-project.properties
-sonar-runner
+# Invoke SonarScanner. This reads from sonar-project.properties
+sonar-scanner
 ```
 
 ## Dependencies with vulnerabilities
@@ -41,4 +41,4 @@ Permission to modify and redistribute is granted under the terms of the [LGPLv3]
   [dependency-check]: https://www.owasp.org/index.php/OWASP_Dependency_Check
   [npm]: https://www.npmjs.com/get-npm
   [dependency-check-cli]: https://jeremylong.github.io/DependencyCheck/dependency-check-cli/index.html
-  [sonar-runner]: https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner
+  [sonar-scanner]: https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner

--- a/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/DependencyCheckSensor.java
+++ b/sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/DependencyCheckSensor.java
@@ -46,7 +46,7 @@ public class DependencyCheckSensor implements ProjectSensor {
 
     private static final Logger LOGGER = Loggers.get(DependencyCheckSensor.class);
     private static final String SENSOR_NAME = "Dependency-Check";
-    private static final String XSD = "https://jeremylong.github.io/DependencyCheck/dependency-check.2.2.xsd";
+    private static final String XSD = "https://jeremylong.github.io/DependencyCheck/dependency-check.2.3.xsd";
 
     private final FileSystem fileSystem;
     private final PathResolver pathResolver;

